processors/HNL.py

```diff
@@ -203,6 +194,65 @@
        leptonCollection = lambda event: event.tightMuon[0] 
      )
 )
+
+analyzerChain.append(
+    JetMetUncertainties(
+	era = globalOptions["year"],
+	globalTag=jecTags[globalOptions["year"]],
+	jerTag = jerTags[globalOptions["year"]],
+	isData= args.isData 
+    )
+)
+
+
+if isMC:
+	
+	for systName,collection in [
+        	("nominal",lambda event: event.jets_nominal),
+	        ("jerUp",lambda event: event.jets_jerUp),
+	        ("jerDown",lambda event: event.jets_jerDown),
+	        ("jesTotalUp",lambda event: event.jets_jesUp["Total"]),
+	        ("jesTotalDown",lambda event: event.jets_jesDown["Total"]),
+    	]:
+
+        	analyzerChain.append(
+            		JetSelection(
+                		inputCollection=collection,
+				leptonCollection=lambda event: event.tightMuon,		
+	                	outputName="selectedJets_"+systName,
+		                storeKinematics=['pt','eta'],
+        			globalOptions=globalOptions
+			)
+	        )
+
+	
+		analyzerChain.append(
+			EventObservables(
+			jetCollection = collection,
+			leptonCollection = lambda event: event.tightMuon[0],
+			outputName="Jet_Muon_"+systName
+			)
+		)
+	
+	for systName,jetCollection,metObject in [
+        	("nominal",lambda event: event.selectedJets_nominal,lambda event: event.met_nominal),
+	        ("jerUp",lambda event: event.selectedJets_jerUp,lambda event: event.met_jerUp),
+	        ("jerDown",lambda event: event.selectedJets_jerDown,lambda event: event.met_jerDown),
+	        ("jesTotalUp",lambda event: event.selectedJets_jesTotalUp,lambda event: event.met_jesUp["Total"]),
+	        ("jesTotalDown",lambda event: event.selectedJets_jesTotalDown,lambda event: event.met_jesDown["Total"]),
+        	("unclEnUp",lambda event: event.selectedJets_nominal,lambda event: event.met_unclEnUp),
```

Comment:
what is here?

Reply:
the unclEnUp

Reply:
it is the uncertainty due to the unclustered jets below 15 GeV there is also the down one